packages/react-mutation-mapper/src/component/mutationMapper/FilterResetPanel.tsx

```diff
@@ -7,6 +7,7 @@ import styles from './filterResetPanel.module.scss';
 type FilterResetPanelProps = {
     resetFilters: () => void;
     filterInfo?: JSX.Element | string;
+    shiftClickMessage?: JSX.Element | string;
```

Comment:
I would name this prop `additionalInfo` or something like that. `FilterResetPanel` doesn't need to know what a `shift+click` is.